src/main/java/org/ngafid/ProcessUpload.java

```diff
@@ -51,13 +44,24 @@
 import org.ngafid.accounts.Fleet;
 import org.ngafid.accounts.User;
 
-import static org.ngafid.flights.DJIFlightProcessor.processDATFile;
+
+@FunctionalInterface
+interface FlightFileProcessors {
```

Comment:
Not in love with the `Object...`. We can probably do this in a cleaner way.

Reply:
The move is probably going to be to move to using `FlightFileProcessor::create`, which takes in the `ZipEntry` + creates the `InputStream`.